src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -460,6 +460,26 @@ static Library()
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: SingleArgTrig(x => x * 180.0 / Math.PI))
             },
+            {
+                BuiltinFunctionsCore.Dec2Hex,
+                StandardErrorHandling<NumberValue>(
+                    expandArguments: InsertDefaultValues(outputArgsCount: 2, fillWith: new BlankValue(IRContext.NotInSource(FormulaType.Blank))),
+                    replaceBlankValues: ReplaceBlankWithZero,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
+                    checkRuntimeValues: FiniteChecker,
```

Comment:
>FiniteChecker [](http://example.com/codeflow?start=40&length=13)

No need to use the FiniteChecker anymore - it is applied to all functions. We can use DeferRuntimeValueChecking now